agreement.md

```diff
@@ -1,32 +1,39 @@
 # RoboJackets Membership Agreement
 
-## Participant Release and Hold Harmless Agreement
+This agreement describes expectations for your behavior while on the Student Competition Center grounds, traveling with
+RoboJackets, or otherwise representing RoboJackets. Failure to comply with this agreement may result in a temporary or
+permanent suspension of membership and/or access to the Student Competition Center, at the discretion of RoboJackets,
+the Georgia Institute of Technology, or their agents.
 
-This Release and Hold Harmless agreement pertains to {{ $full_name }} (“Participant”) and the Participant’s involvement
+This agreement does not obligate you to pay dues to RoboJackets; however, you must sign it before we will accept payment.
+
+## RoboJackets, Inc. Participant Release and Hold Harmless Agreement
+
+This Release and Hold Harmless agreement pertains to {{ $full_name }} ("Participant") and the Participant's involvement
 in activities and travel related to RoboJackets, Inc., a nonprofit corporation organized and existing under the laws of
 the State of Georgia.
 
 @if(!$is_electronic)
-If Participant is less than 18 years of age or has a legal guardian, then this release is signed by Participant’s parent
-or legal guardian (such parent or legal guardian being referred to herein after as “Guardian”), on behalf of and binding
-on Guardian and Participant, and Guardian gives permission for Participant to participate and engage in all activities
-related to RoboJackets.
+If Participant is less than 18 years of age or has a legal guardian, then this release is signed by Participant's parent
+or legal guardian (such parent or legal guardian referred to hereinafter as "Guardian"), on behalf of and binding on
+Guardian and Participant, and Guardian permits Participant to participate and engage in all activities related to
+RoboJackets.
 @endif
 
 1. **Waiver and Release:** I, the Participant, hereby release, to the fullest extent permitted by law, RoboJackets, Inc.
-and all of its directors, officers, and agents (herein referred to collectively as “RoboJackets”) from any and all
-liability, claims, demands, injuries, damages and causes of action arising from or relating to my participation in
+and all of its directors, officers, and agents (herein referred to collectively as "RoboJackets") from any and all
+liability, claims, demands, injuries, damages, and causes of action arising from or relating to my participation in
 activities and travel in connection with RoboJackets.
```

Comment:
This is missing the new "Damage Waiver" clause I wrote yesterday.

> Damage Waiver: In no event shall RoboJackets be liable under this agreement for any special, consequential, or incidental damages of any kind arising from my participation in activities and travel in connection with RoboJackets.

Reply:
This is included in lines 27-28.